associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -226,20 +227,20 @@
     <!-- Macro responsável por mostrar o autor na citação -->
     <choose>
       <if type="broadcast motion_picture song" match="any">
-        <text variable="title" form="short" text-case="uppercase" quotes="false"/>
+        <text variable="title" form="short" text-case="capitalize-first" quotes="false"/>
```

Comment:
Is there any reason you're using 'capitalize-first' for all the titles and author names? In 99% of the cases, these will be properly stored in the database, including in the rare cases where authors strongly prefer their names to be lowercased (danah boyd, e.g.)
I'd suggest removing the text-case attribute entirely throughout.

Reply:
Thanks, @adam3smith ! Attribute removed.

Reply:
Actually, I'll have to make some changes on this issue, but I don't know how to do it.

How can I make a `name-part name="family"` and a substitute clause when there is no author and we use title be uppercase only at bibliography?

```
			<else-if type="entry-dictionary entry-encyclopedia" match="any">
				<names variable="author">
					<!-- para outros tipos de documentos mantém a posição original -->
					<name delimiter="; " delimiter-precedes-et-al="never" delimiter-precedes-last="always" name-as-sort-order="all">
						<name-part name="family"/>
						<name-part name="given"/>
					</name>
					<et-al font-style="italic"/>
					<label form="short" text-case="lowercase" prefix=" (" suffix=".)"/>
					<substitute>
						<!-- na ausência de um autor, utiliza organizador, editor, tradutor ou título  -->
						<text variable="title" />
					</substitute>
				</names>
			</else-if>
```